### ISSUE TITLE
[BUG] DJ018 not catching data-src urls

### ISSUE THREAD

**Issue by christopherpickering** [djlint/djLint#116]
rule D/J018 should be catching this:

```html
<div class="em-ajaxLogs" data-src="/table/task/{{ t.id }}/log">
```

**Comment by christopherpickering** [comment]
also ``action`` on an input
